pkg/virt-handler/vm.go

```diff
@@ -448,6 +450,10 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 	} else if vmi.Status.NodeName != "" && vmi.Status.NodeName != d.host {
 		// Only update the VMI's phase if this node owns the VMI.
 		// not owned by this host, likely the result of a migration
+
+		// Clear the pod network cache files now because we're sure VMI doesn't exist here anymore
```

Comment:
You sould verify `isPreMigrationTarget` is not true. So you won't cleanup the prepared by the target during a migration.